test/pytest/test_xpedite/data/dataTxnAppRegularManifest.csv

```diff
@@ -0,0 +1,10 @@
+file name, size, lines
+dataTxnAppRegular/__init__.py,0,0
```

Comment:
`__init__.py` here as well. is it needed ?

Reply:
I think so, to import profileInfo from the regular scenario in the benchmark scenario
from dataTxnAppRegular.parameters.profileInfo import *